frontend/src/metabase/query_builder/components/notebook/Notebook.jsx

```diff
@@ -26,12 +27,18 @@ export default function Notebook({ className, ...props }) {
           primary
           style={{ minWidth: 220 }}
           onClick={async () => {
-            let cleanQuestion = question.setQuery(question.query().clean());
-            if (cleanQuestion.display() === "table") {
-              cleanQuestion = cleanQuestion.setDefaultDisplay();
+            // Only update the question if it's dirty, otherwise Metabase
```

Comment:
This may be a great opportunity to move out the function currently inside the `onClick` prop.

So it would look roughly like 

```js
...
} = props;

function cleanupQuestion() {
…
}

return (
 ...
 onClick={cleanupQuestion}
 ...
)
```

Reply:
Thanks @gus